docs/source/setup_lora.rst

```diff
@@ -21,152 +21,151 @@ first be created at TTN (if one does not already exist).
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-If you want the MultiGeiger to pass recorded data to *sensor.community*
-via TTN, you have to register it. The registration is similar to the TTN
-registration described above. In the following, only the changes are
+Logging data to sensor.community
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you want to transfer the data from Multigeiger to sensor.community, you have to 
+register at sensor.community. The registration process is similar to the description 
+above ("Login to sensor.community").  In the following, only the changes are
 explained:
 
--  Sensor ID:
+**Sensor ID:**
    Enter the last 4 bytes of the DEVEUI in left to right order (e.g. if
    the DEVEUI is *00 D0 C0 00 C3 19 7C E8*, so enter *C3197CE8*), but
    converted to decimal, not in HEX (finally: 3273227496).
--  Sensor Board:
-   Select **TTN**, using the small arrows on the right.
-
-HTTP integration
-~~~~~~~~~~~~~~~~
-
-To get the data from TTN to *sensor.community* you have to enable the
-HTTP integration at TTN. In the TTN console click *Applications* and
-then click on the application of the GeigerCounter
-(e.g. *geiger_20200205*). On the top right in the bar with *Overview*,
-*Devices*, *Payload Formats*, *Integrations*, *Data*, *Settings* click
-**Integrations**. Then select **HTTP Integration** via **add
-integration**.
-
-Now fill in the displayed fields:
- - **Process ID**
-   Enter any name for this integration here **Access Key:**
-   Click here once and select the *default key*
- - **URL:**
-   Enter the URL for the ttn2luft program: https://ttn2luft.citysensor.de
- - **Method:**
-   If it reads already *POST*, don’t touch it
- - **Authorization:** 
-   remains empty
- - **Custom Header Name:**
-   here comes the text **X-SSID** pure
- - **Custom Header value:**
-   Enter the SSID of the sensor (the number you got when you registered at sensor.community, *NOT* the chip ID).
- - Click **Add integration** in the lower right corner to confirm the   changes.
-
-See this example how the form should look like:
-|“HTTP Integration filled”|
+**Sensor Board:**
+   Select **TTN**, to select use the small arrow on the right.
+
+Webhook integration
+~~~~~~~~~~~~~~~~~~~~
+
+To get the data via TTN to *sensor.community*, the Webhook integration at TTN has to be activated.
+
+At the Application tab select **Integrations** from the left menue. Select **Webhooks**, then **+ Add webhook**. 
+Scroll down and select **Custom webhook**.
+
+Fill in the following fields:
+
+**Webhook ID**
+  Enter an arbitrary name for this hook
+**Webhook format**
+  Select **JSON**
+**Base URL**
+  Enter **https://ttn2luft.citysensor.de**
+**Downlink API key**
+  This remains empty
+
+Then click at **+ Add header entry** to add a special Header. Enter **X-SID** in the left field and your 
+sensor id (the number, you received from sensor.community, **not** the chip ID) in the right field.
```

Comment:
isn't this describing the same as the "webhook_filled" picture is showing?

if the pic is superfluous, remove it.